projects/dsp-ui/ng-package.json

```diff
@@ -7,7 +7,8 @@
       "@knora/api": "@knora/api",
       "ngx-color-picker": "ngxColorPicker",
       "jdnconvertiblecalendar": "jdnconvertiblecalendar",
-      "jdnconvertiblecalendardateadapter": "jdnconvertiblecalendardateadapter"
+      "jdnconvertiblecalendardateadapter": "jdnconvertiblecalendardateadapter",
+      "ts-md5": "Md5"
```

Comment:
Shouldn't this be "tsMd5" on the right?

Reply:
Both seem to work but I changed it to "tsMd5" in 91d6b27

Reply:
maybe you can put what you want, I don't know. "tsMd5" is just what the compiler guessed.